sdk/src/aleo_network_client.ts

```diff
@@ -250,6 +262,88 @@ export class AleoNetworkClient {
     }
   }
 
+  /**
+   * Returns the proving and verifying keys for a specified function in an Aleo Program given the url of the prover and
+   * verifier keys
+   *
+   * @param {string} url of the proving key
+   * @param {string} url of the verifying key
+   *
+   * @returns {Promise<[ProvingKey, VerifyingKey] | Error>} Proving and verifying keys for the specified program
```

Comment:
JSDoc complains when you use brackets or curly braces.
Do you mind escaping them in the new docs?
```suggestion
   * @returns {Promise<\[ProvingKey, VerifyingKey\] | Error>} Proving and verifying keys for the specified program
```

Reply:
Actually escaping via the suggestion I made does not work either hmm...

Reply:
@collinc97 not sure if something like this would be worth it but [TypeDoc](https://typedoc.org) might be useful to solve issues like this. It'll automatically pull the types which prevents kind of redundant work and the possibility of the docs getting out of sync with the code.

Reply:
I could potentially decouple these methods, however since the verifying key is so much smaller than the proving key and I'm trying to encourage always using them in pairs (rather than encouraging disparate keys accidentally being used) I'm inclined to keep them this way. However, perhaps we could update the method signature

Reply:
I just created a type Signature for it, and that solved the issue